qa/tasks/mgr/dashboard/helper.py

```diff
@@ -646,7 +646,7 @@ def __new__(cls, sub_elems, allow_unknown=False, none=False, unknown_schema=None
         'dev': str,
         'path': str
     }))
-}))
+}, allow_unknown=True))
```

Comment:
Possible Conflict with this PR https://github.com/ceph/ceph/pull/39596/files

Reply:
@nizamial09 how can a change backported from master conflicts with its original change?